resources/profiles/Anker.ini

```diff
@@ -26,156 +26,817 @@ default_materials = Generic PLA+ @ANKER; Generic PLA @ANKER; Generic PET @ANKER;
 
 # All presets starting with asterisk, for example *common*, are intermediate and they will
 # not make it into the user interface.
+[printer_model:M5C]
+    name = AnkerMake M5C
```

Comment:
Please remove the leading spaces. This is inconsistent with other configurations and makes it difficult to review as it is hard to see what values actually changed.

Reply:
OK

Reply:
I still see the leading spaces in this area.